<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Help</class>
 <widget class="QDialog" name="Help">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>449</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>473</width>
    <height>449</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>473</width>
    <height>449</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Aide</string>
  </property>
  <widget class="QFrame" name="frame_image">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>81</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbl_image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pas-à -pas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_prev">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>81</y>
     <width>31</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>◄</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_next">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>81</y>
     <width>31</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>►</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_desc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbl_desc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="b_close">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_step">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Etape</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>b_close</sender>
   <signal>clicked()</signal>
   <receiver>Help</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
